_posts/en/newsletters/2020-06-10-newsletter.md

```diff
@@ -0,0 +1,311 @@
+  network and delays the relay of new block announcements to the victim.
+  After the victim's view of the block chain is far enough behind the
+  public consensus view, the attacker closes their channels with the
+  victim in an outdated state that pays the attacker more than the
+  latest state would.  The eclipse attack is used to prevent the victim
+  from seeing the close transactions until after the dispute period is
+  over and the attacker has withdrawn their illicit gains to an address
+  entirely under the attacker's control.
+
+    The paper describes an attack that can steal from lightweight
+    clients in as little as two hours.  For attacks against LN nodes
+    backed by a local Bitcoin full verification node, their shortest
+    duration attack requires eclipsing the victim for at least 24 hours.
+    The authors note that the attacks probably apply to other
+    time-sensitive contract protocols and that the best solutions to the
+    problem all involve improving eclipse attack resistance.
+
+- **Fee overpayment attack on multi-input segwit transactions:** The
+  transaction fee for a Bitcoin transaction is the difference between
+  the amounts of all the UTXOs that transaction spends and the amounts
+  of all the UTXOs it creates.  Transactions explicitly state the amounts
+  of the UTXOs they create, but the amounts of the UTXOs they spend can
+  only be found by looking at the previous transactions that created
+  those UTXOs.  The spending transaction simply commits to the txid and
+  location of the UTXOs it wants to spend, requiring other software that
+  wants to calculate the fee to either look up each UTXO's previous
+  transaction or to maintain a database of verified UTXO data.
+
+    Hardware wallets don't maintain a UTXO set, so the only trustless
+    way for them to determine the amount of fee paid by a transaction
+    with legacy UTXOs is to get a copy of each legacy UTXO's previous
+    transaction, hash the previous transaction to ensure its txid
+    matches the UTXO reference, and use the now-verified amount of the
+    UTXO to perform the fee calculation.  Since legacy transactions can
+    be almost as large as one megabyte and it's possible for a spending
+    transaction to reference thousands of previous transactions, this
+    process can require the processing of gigabytes of data by
+    resource-constrained hardware wallets.
+
+    One of the several improvements made in [BIP143][] segwit v0 was an
+    attempt to eliminate this burden by having [signatures commit to the
+    amount][bip143 motivation] of the UTXO they were spending.  This
+    meant that any signatures that committed to an incorrect amount
+    would be invalid, which developers of both [Bitcoin Core][towns
+    benefits] and [hardware wallets][palatinus inpatient] believed would
+    allow signers to safely accept amounts from untrusted programs.
+
+    Unfortunately, signing a single UTXO's amount turned out to be
```

Comment:
I'd emphasize that the 'attacker' is not able to directly steal any funds from the victim, simply make him burn bitcoin in fees.